### DISCUSSION TITLE
Neuroanatomy detailed atlas program

### DISCUSSION THREAD

**Discussion by eliasdally2** [community/community#147040] [Programming Help]
### Body

I am a 5th year medical student, interested in creating a program showing the 3D structure of the brain, showcasing all the different connections between neurons, synapses, and their termination. 
I don't really know coding, but I am interested in finding someone who is, and trying to work together to create such a project. 
The human brain is extremely difficult to comprehend, especially with the numerous amount of connections between the neurons and the various centres present in it.
Unfortunately, today most of the software available for students is just showing the gross anatomy, just like how the brain is seen to the naked eye, not really showing individual tracts origin and termination. 
Creating such an application will help students greatly!  
I really don't know where to post such as idea, but if you are interested, feel free to reach out! I have some nice ideas that I would like to share. 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mecodeatlas** [comment]
Hey @eliasdally2!

Your idea is super interesting and could be really helpful for students! Have you seen any existing programs that try to do something similar, even if they’re not great? It could help people understand what you’re aiming for. Also, do you already have some specific features in mind, or is it still more of a general idea? Feel free to share :)

**Comment by eliasdally2** [comment]
I was looking for such features for a long time in a program, wasted so much money on many programs, but unfortunately couldn't find the thing I am looking for. 
It's really difficult for us, in the medical field, to explain exactly what we want from programmers, and I guess that's the problem these programs are having. 
In a nutshell, the CNS (central nervous system) has a very complex structure, but luckily, it's a continuous structure, so if structure X was in the front of the brain, and becomes at the back of the spinal cord, then at some point, structure X had a path to travel from front to back. 
What makes neuroanatomy especially difficult is imagining how it got back, at what level, and next to what other structures; knowing this information is particularly helpful in pinpointing the exact location where an abnormality can be found/originating from (like a stroke). 
My main idea is to create a 3D model of the brain, connected to the spinal cord, and mapping out the entire neuronal map, pathways going up (sensations) and pathways going down (motor functions). 
To our bad luck as medical students, there are multiple pathways, carrying different modalities to the brain, each of them in a different tract (different location in the spinal cord, different route [location of crossing to the other side], and so on ...). 
My plan is to create this map, showing each tract in a different color, and have some explanation for each of them, highlighting the important landmarks the pathway comes across. 
Thank you for your interest! 
Elias
